src/views/PeopleSearch/index.js

```diff
@@ -425,14 +427,27 @@ class PeopleSearch extends Component {
     this.setState((state) => ({ nameExpanded: !state.nameExpanded }));
   };
   handleAcademicsExpandClick = () => {
+    this.setState({
+      homeExpanded: false,
+      offDepExpanded: false,
+    });
     this.setState((state) => ({ academicsExpanded: !state.academicsExpanded }));
```

Comment:
A small thing, but I see no reason why these should be split across two setStates. They should work fine (and be more easily readable) as one unified setState call.